units/patch_intent.go

```diff
@@ -992,8 +992,18 @@ func (j *patchIntentProcessor) buildGithubPatchDoc(ctx context.Context, patchDoc
 		})
 	}
 
+	j.user, err = findEvergreenUserForPR(patchDoc.GithubPatchData.AuthorUID)
+	if err != nil {
+		return isMember, errors.Wrapf(err, "finding user associated with GitHub UID '%d'", patchDoc.GithubPatchData.AuthorUID)
+	}
+	patchDoc.Author = j.user.Id
+	patchDoc.Project = projectRef.Id
+
 	patchContent, summaries, err := thirdparty.GetGithubPullRequestDiff(ctx, githubOauthToken, patchDoc.GithubPatchData)
 	if err != nil {
+		if strings.Contains(err.Error(), thirdparty.PRDiffTooLargeErrorMessage) {
+			return isMember, nil
```

Comment:
If we're going to skip updating `Patches` here, can we set something on the patch document that indicates the diff was too large for us? This could be helpful [DEVPROD-5879](https://jira.mongodb.org/browse/DEVPROD-5879) when we display an error that the diff was too large.

Reply:
Adding something here could potentially break the UI because it would also require a summary. Adding a flag for diff too large could be future work but I didn't think it was worth doing now.

I was thinking the UI update would say something like "No code changes or diff too large to show" 
because this is a quick fix and sever wanted an urgent fix.

Reply:
unless you feel strongly

Reply:
Since this is urgent it's fine but ideally we should distinguish between empty patches and patches that have diffs that are too large, so we should ticket that out.

Reply:
created! https://jira.mongodb.org/browse/DEVPROD-5926